_test/json/schema1.struct.json

```diff
@@ -2,6 +2,7 @@
     "structversion": "2016-03-21",
     "schema": "schema1",
     "id": "1",
+    "user": "",
```

Comment:
Should this be there? I'd think we shouldn't export nor import the user into/from the json files.